### PR TITLE
Thorntail-runner: ClassNotFoundException in multi-module package

### PR DESCRIPTION
**Pull request by bojanb1234**
Description is provided on an isue:  https://github.com/thorntail/thorntail/issues/1276

### CODE REVIEW

**Comment by Ladicek** [comment]
ok to test

**Comment by Ladicek** [comment]
ok to test

**Comment by Ladicek** [comment]
Bump @michalszynkiewicz

File: thorntail-runner/src/main/java/org/wildfly/swarm/runner/WarBuilder.java [hunk 1]
```diff
@@ -159,32 +159,29 @@ private void addClassesToZip(ZipOutputStream output, String prefix, File classes
...
     }
 
     private void addClassToZip(ZipOutputStream output, String prefix, File file, File classesDirectory) {
+        URI base = classesDirectory.toURI();
         try {
-            String name = getRelativePath(file, classesDirectory.getAbsolutePath());
+            String name = base.relativize(file.toURI()).getPath();
             name = prefix + name;
-            writeFileToZip(output, file, name);
+            if (file.isDirectory()) {
```
Comment (michalszynkiewicz) [inline thorntail-runner/src/main/java/org/wildfly/swarm/runner/WarBuilder.java:173]: Was it failing for you without adding an entry for directory? 
It seems it worked without it so far.

**Review by michalszynkiewicz — APPROVED** [review]

File: thorntail-runner/src/main/java/org/wildfly/swarm/runner/WarBuilder.java (hunk 1 above)
Comment (bojanb1234) [reply thorntail-runner/src/main/java/org/wildfly/swarm/runner/WarBuilder.java:173]: Yes it was failing for me without an entry for a folder. I don't know about linux system, but on windows it was failing.

Comment (michalszynkiewicz) [reply thorntail-runner/src/main/java/org/wildfly/swarm/runner/WarBuilder.java:173]: thanks for clarification

**Pull request merged**